src/test/resources/cartridge/incorrect_topology.lua

```diff
@@ -0,0 +1,16 @@
+cartridge = require('cartridge')
```

Comment:
q: what is incorrect in this topology?

Reply:
There's no server on 3303 port. I added comment in topology